repos/runtime.common.props

```diff
@@ -0,0 +1,145 @@
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:OutputRid=$(OverrideTargetRid)</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:DotNetOutputBlobFeedDir=$(SourceBuiltBlobFeedDir)</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) /p:BuiltSdkPackageOverride=""</InstallerBuildArguments> <!-- Clear PackageOverride for Installer - it only applies to coreclr -->
+      <InstallerBuildArguments>$(InstallerBuildArguments) /m:1</InstallerBuildArguments>
+
+      <InstallerBuildArguments>$(InstallerBuildArguments) $(FlagParameterPrefix)verbosity $(LogVerbosity)</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) $(FlagParameterPrefix)binaryLog</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</InstallerBuildArguments>
+      <InstallerBuildArguments>$(InstallerBuildArguments) $(FlagParameterPrefix)nodereuse $(ArcadeFalseBoolBuildArg)</InstallerBuildArguments>
+
+      <BuildArguments>$(FlagParameterPrefix)arch $(Platform)</BuildArguments>
+      <BuildArguments>$(BuildArguments) $(FlagParameterPrefix)configuration $(Configuration)</BuildArguments>
+      <BuildArguments>$(BuildArguments) $(FlagParameterPrefix)binaryLog</BuildArguments>
+      <BuildArguments>$(BuildArguments) $(FlagParameterPrefix)ci</BuildArguments>
+      <BuildArguments>$(BuildArguments) $(FlagParameterPrefix)runtimeConfiguration $(Configuration)</BuildArguments>
+
+      <BuildCommand>$(ProjectDirectory)build-source-build$(ShellExtension) $(BuildArguments) $(FlagParameterPrefix)coreclr-args $(CoreClrBuildArguments) $(FlagParameterPrefix)libraries-args $(LibrariesBuildArguments) $(FlagParameterPrefix)installer-args $(InstallerBuildArguments)</BuildCommand>
+      <BuildCommand Condition="$(Platform.Contains('arm')) AND '$(BuildArchitecture)' != 'arm64'">$(ArmEnvironmentVariables) $(BuildCommand)</BuildCommand>
+      <CleanCommand>$(ProjectDirectory)/clean$(ShellExtension)</CleanCommand>
+  </PropertyGroup>
+
+  <!-- Output / source-build flags -->
+  <PropertyGroup>
+      <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
+      <PackagesOutput>$(ProjectDirectory)bin/Product/$(TargetOS).$(Platform).$(Configuration)/.nuget/pkg</PackagesOutput>
+      <ShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</ShippingPackagesOutput>
+      <NonShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/NonShipping/</NonShippingPackagesOutput>
+      <NuGetConfigFile>$(ProjectDirectory)/NuGet.config</NuGetConfigFile>
+      <DependencyVersionInputRepoApiImplemented>true</DependencyVersionInputRepoApiImplemented>
+      
+      <!-- core-setup does not destabilize for intermediate servicing builds -->
+      <IsStable>true</IsStable>
```

Comment:
This isn't true in 3.1 anymore, and doesn't seem right to have when trying to build 5.0-p2 either. This is checked into the repo now, since ProdCon v1 and its fancy parameters aren't used anymore.

Reply:
Removed.